tests/diagnostics/uninitialized-globals.slang

```diff
@@ -45,4 +45,34 @@ float use_later_writte()
     return writtenLater;
 }
 
+// Should not warn here (glsl layout decoration check)
+layout(location = 0) in vec4 data;
+
+vec4 glsl_layout_in_ok()
+{
+    return data;
+}
+
+// Invalid use of layout variables
```

Comment:
I don't think this comment describes why we have this test case.

I suggest something like the following:
```suggestion
// Varying outputs should be initialized by the end of a shader.
```